### DISCUSSION TITLE
Stimuli and Participant Capacity

### DISCUSSION THREAD

**Discussion by s1975** [l3atbc-datadog/pushkin#184] [Q&A]
Hello. I was wondering what the capacity is for stimuli files is on the server? For our research we were hoping to be able to have around 1,000 stimuli files possibly more. I also was wondering what the maximum number of participants per experiment can be? We were hoping for a number around 200 as well. Any information on this would be great. Thank you!

**Comment by jkhartshorne** [comment]
Wow, sorry for the slow response. There is no capacity limit for either.
